apps/web/src/components/user/AclMask.tsx

```diff
@@ -1,9 +1,11 @@
 import { PropsWithChildren, ReactNode } from 'react';
 import { Access, hasAccess } from '@ien/common';
 import { useAuthContext } from '../AuthContexts';
+import * as React from 'react';
```

Comment:
do we need this line?

Reply:
It's why we shouldn't do `copy and paste`